framework/lib/components/text-field/formatted-number-input.tsx

```diff
@@ -78,46 +79,65 @@ export const FormattedNumberInput = ({
   numberOfDecimals = 2,
   max = Infinity,
   min = -Infinity,
+  inputLeftElement,
+  inputRightElement,
   ...rest
 }: FormattedNumberInputProps) => {
   const props = presetMap[preset]
+  const [ v, setV ] = useState(value)
```

Comment:
Why do we need a state here instead of using the value?
Seems pretty risky to shadow the value in this case

Reply:
@martinmartinchan Yes, but it is the only solution I found to be able to set/change the value when the inputed value goes above the max or below min